qhub/provider/cicd/common.py

```diff
@@ -0,0 +1,13 @@
+import os
+
+
+def pip_install_qhub(qhub_version: str) -> str:
+    qhub_gh_branch = os.environ.get("QHUB_GH_BRANCH")
```

Comment:
what happens if there is no ENV for whatever reason? need to provide a fallback or at least raise an error here

Reply:
If `os.environ.get` can't find the specified env var, it will default to None. That said, I will make this explicit :)